src/Rx/Weno/logsync.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
this script should be moved to interface subdirectory.
Would put it in:
interface/weno/

Reply:
done